fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/flight_model.cfg

```diff
@@ -606,7 +606,7 @@ cruise_speed = 487.5 ; Knots True (KTAS)
 cruise_mach = 0.85
 crossover_speed = 320 ; Knots Indicated (KIAS)
 max_mach = 0.89 ; MMO
```

Comment:
The max design mach is 0.96, and max design speed is 375kts. So with 390 we have 15kts of buffer even above the Vd, which is probably reasonable, as IIRC, the sim immediately starts flutter above these speeds. We should also adjust max_mach to be above Md with a similar buffer, I think Vd+0.01 is reasonable, so I'd say max_mach=0.97